jbmc/src/java_bytecode/java_class_loader_limit.cpp

```diff
@@ -13,8 +13,35 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <json/json_parser.h>
 
-/// initializes class with either regex matcher or match set
-/// \par parameters: parameter from `java-cp-include-files`
+/// Initializes class with either regex matcher or match set. The regex match
```

Comment:
_Shudders at how this works_. Could you make the second sentence say something like:
```C++
If the string starts with an @ it is treated as a path to a json file. Otherwise, it is treated as a regex. 
```

before continuing with the explanation of the two different formats.